react/client/src/components/formPages/BeforeYouBegin.tsx

```diff
@@ -1,20 +1,44 @@
 import React from 'react';
 
-import { Wrapper } from '../../styles/BeforeYouBegin'
+import arrowRight from '../../assets/arrowRight.svg'
+
+import Button from 'components/Button';
+
+import { Flex } from 'styles/GlobalStyle'
+
+// this is kind of funky but will need to be updated when text is changed
+const disclaimerText = `
+Please make sure you are eligible for getting your record cleared. If you need help with this please email placeholder@website.com.
+
+Be aware your statement is stored locally on your device, and never transmitted to our servers or any third-party
+
+This software is developed by a non-governmental third party, Code for America
+
+Please also review our Privacy Policy and Terms of Use before beginning.
+
+Please allocate at least 30 minutes of time to complete this.
+
+If at anytime you are confused please click the question mark button for guidance.
+`
 
 const BeforeYouBegin: React.FC<GlobalProps> = ({ goToPage }) => {
   return (
-    <Wrapper>
-      <h2>Before you begin</h2>
-      <p>Please make sure you are eligible for getting your record cleared. If you need help with this please email placeholder@website.com.</p>
-      <p>Be aware your statement is stored locally on your device, and never transmitted to our servers or any third-party</p>
-      <p>This software is developed by a non-governmental third party, Code for America</p>
-      <p>Please also review our Privacy Policy and Terms of Use before beginning.</p>
-      <p>Please allocate at least 30 minutes of time to complete this.</p>
-      <p>If at anytime you are confused please click the question mark button for guidance.</p>
```

Comment:
Need to re-explore this a bit. Semantically, these aren't paragraphs dividing text but I might have gone too far the other way to remove the `<p>` tag completely.